src/openforms/forms/tests/test_data_migrations.py

```diff
@@ -1,7 +1,13 @@
 from unittest import skip
 
+from django.apps import apps
+from django.db import IntegrityError
```

Comment:
unused import

Reply:
apps is required as argument to the migration function. The IntegrityError was indeed a leftover from a previous stab at the problem.